### PR TITLE
html form

### PR DESCRIPTION
**Pull request by ISRLv2021**
- [DEMO LINK](https://ISRLv2021.github.io/layout_html-form/)
- [TEST REPORT LINK](https://ISRLv2021.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,152 @@
...
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname" class="label">
+          Surname:
+          <input
+            type="text"
+            id="surname"
+            class="input"
+          >
+        </label>
```
Comment (Denchepornyuk) [inline src/index.html:28]: you can remove `for` and `id` attributes because input is inside the label, fix in all places

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,152 @@
...
+          <input
+            type="number"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+            class="input"
+          >
+        </label>
+        <label for="dateOfBirth" class="label">
```
Comment (Denchepornyuk) [inline src/index.html:48]:
```suggestion
        <label for="date-of-birth" class="label">
```

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,152 @@
...
+            class="input"
+          >
+            Yes
+          <input
+            type="checkbox"
+            value="1"
+            class="input"
+          >
+            No
+        </label>
```
Comment (Denchepornyuk) [inline src/index.html:103]: don't use one label for two inputs, create separate labels for each input

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,152 @@
...
+        <label for="cars" class="label">
+          What are your favorite brand of cars?
+          <select
+            id="cars"
+            multiple
+            required
+          >
+            <option value="BMW">BMW</option>
+            <option value="BMW">Audi</option>
+            <option value="BMW">BMW</option>
```
Comment (Denchepornyuk) [inline src/index.html:121]: Why are values `BWM` for every option?

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (ISRLv2021) [reply src/index.html:28]: ok, thank you a lot

File: src/index.html (hunk 3 above)
Comment (ISRLv2021) [reply src/index.html:103]: I did it, but with one extra css class

File: src/index.html (hunk 4 above)
Comment (ISRLv2021) [reply src/index.html:121]: got it, thanks

File: src/index.html [hunk 5]
```diff
@@ -11,7 +11,147 @@
...
+        <legend>An interesting fact about you!</legend>
+        <div class="label">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="yes"
+            >
+                Yes
```
Comment (Denchepornyuk) [inline src/index.html:88]:
```suggestion
              Yes
```

File: src/index.html [hunk 6]
```diff
@@ -11,7 +11,147 @@
...
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="no"
+            >
+                No
```
Comment (Denchepornyuk) [inline src/index.html:96]:
```suggestion
              No
```

**Review by Denchepornyuk — APPROVED** [review]

File: src/index.html (hunk 6 above)
Comment (ISRLv2021) [reply src/index.html:96]: Understandable, thank you

**Pull request closed without merging**